bin/devtools.dart

```diff
@@ -42,6 +42,10 @@ void main(List<String> arguments) async {
   }
 
   machineMode = args[argMachine];
+
+  final buildFolder = await Isolate.resolvePackageUri(
```

Comment:
It looks like this returns a `Uri`. Is this always a `file:` uri? If so, we could convert that to a file on disk, which would resolve into the `lib/` directory. We could then travel up to the parent dir and down to `build/`.

Reply:
I was going to do that originally but it felt a bit weird (not least because you can't resolve `package:devtools/` we have to add something to the end, so I ended up with `package:devtools/blah` and then traversed up).

I also wasn't sure whether there were any guarantees made about the location of things outside of lib, so figured this may be safer. If we're going to build the package into a separate folder from a build script (which I think we should, so we can make sure it has exactly what we want and nothing else, and so we can strip the pubspec deps down) then building into lib probably has no drawbacks.

But, I know very little about pub packages, so I'll go with whatever you think will work best. I need to rebase this since the move so I might extract the build steps (since there are now a few) into a shell script for now to make it clearer what's going on (and easier to run outside of VS Code).